ngraph/src/ngraph/frontend/onnx_import/exceptions.hpp

```diff
@@ -34,22 +34,6 @@ namespace ngraph
                 std::string get_error_msg_prefix(const Node& node);
             }
 
-            struct NotSupported : AssertionFailure
```

Comment:
Why are we removing these more specific exceptions are replacing them with the less specific `ngraph_error`?

Reply:
In fact those specific exceptions haven't been adopted widely in our code. They were used only once each. Moreover we're not utilizing their specific type to introduce more specific error message nor any kind of special handling. Now when we're wrapping thrown errors and using `CHECK_VALID_NODE` macro I think we don't need them.